concepts/review.md

```diff
@@ -1,5 +1,103 @@
-# Work in Progress
+# Getting Analysis Review
+
+The current process for getting analysis review has not been optimized.
+If you run into problems, feel free to ask #datapipeline
+
+## Table of Contents
+
+<!-- toc -->
+
+# Finding a Reviewer
+
+There are two major types of review for most analyses:
```

Comment:
Maybe we should mention "code review" as well? There have been cases in the past where the code review was done by an engineer and the data review by an analyst.